src/contents/news/news.yml

```diff
@@ -1,3 +1,108 @@
+- posted: July 11, 2023
+  image: src/images/thumb/spaceship.png
+  featured: true
+  markdown: |
+      Our submission to the (2023 PACE Challenge)[https://pacechallenge.org/2023/] which involved making solvers for the twinwidth problem has placed first in the exact category as well as receiving the Theory Award! (See results)[https://pacechallenge.org/2023/results/].
+- posted: July 5, 2023
+  image: src/images/thumb/papers_thumb.jpg
+  featured: true
+  markdown: |
+      A preprint of our paper on Meta-analysis of metagenomes is available (on bioRxiv)[https://www.biorxiv.org/content/10.1101/2022.06.30.498290v1].
+- posted: July 3, 2023
+  image: src/images/thumb/papers_thumb.jpg
+  featured: true
+  markdown: |
+      Our paper on (Hetnet connectivity search provides rapid insights into how two biomedical entities are related)[https://www.biorxiv.org/content/10.1101/2023.01.05.522941] has been accepted for publication at GigaScience!
+- posted: July 1, 2023
+  image: src/images/thumb/papers_thumb.jpg
+  featured: true
+  markdown: |
+      Our paper on (Parameterized Complexity of Gerrymandering)[https://arxiv.org/abs/2205.06857] has been accepted for publication at The 16th International Symposium on Algorithmic Game Theory (SAGT)! It will be presented in early September at the University of London.
+- posted: July 1, 2023
+  image: src/images/people/bdsullivan.jpg
+  featured: true
+  markdown: |
+      Dr. Sullivan has become a full professor at the University of Utah. Congratulations!
+- posted: May 28, 2023
+  image: src/images/thumb/arxiv.jpg
+  featured: true
+  markdown: |
+      A preprint of our paper on Edge-Colored Cluster in Hypergraphs is available [on ArXiv](https://arxiv.org/abs/2305.17598).
+- posted: May 15, 2023
+  image: src/images/thumb/papers_thumb.jpg
+  featured: true
+  markdown: |
+      Our paper on (Reducing Access Disparities in Networks Using Edge Augmentations)[https://arxiv.org/abs/2209.07616] has been accepted for publication at the Proceedings of the 2023 ACM Conference on Fairness, Accountability, and Transparency (FAccT '23)! It will be presented at the conference in June.
```

Comment:
```suggestion
      Our paper on (Reducing Access Disparities in Networks Using Edge Augmentations)[https://arxiv.org/abs/2209.07616] has been accepted for publication at the Proceedings of the 2023 ACM Conference on Fairness, Accountability, and Transparency (FAccT '23)! It will be presented at the conference by our recent PhD Ashkan Bashardoust in June.
```